<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>651</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>681</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_len">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_download">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_picture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>541</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_author">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_view">
       <property name="text">
        <string>Views</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_length">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>541</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_mime">
       <property name="text">
        <string>Video type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_channelname">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_title">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_resolution">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_filesize">
       <property name="text">
        <string>Filesize</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
